release-notes/3.13.0.md

```diff
@@ -240,21 +240,6 @@ Release notes are kept under [rabbitmq-server/release-notes](https://github.com/
 
    GitHub issue: [#8218](https://github.com/rabbitmq/rabbitmq-server/pull/8218)
 
- * Reduced memory footprint, improved memory use predictability and throughput of classic queues (version 2, or CQv2).
-   This particularly benefits classic queues with longer backlogs.
-
-   Classic queue v2 (CQv2) storage implementation **is now the default**. It is possible to switch
-   the default back to CQv1 using `rabbitmq.conf`:
-
-   ``` ini
-   # uses CQv1 by default
-   classic_queue.default_version = 1
-   ```
-
-   Individual queues can be declared by passing `x-queue-version` argument and/or through a `queue-version` policy.
-
-   GitHub issue: [#8308](https://github.com/rabbitmq/rabbitmq-server/pull/8308)
-
```

Comment:
Maybe this should be replaced with a note about deprecation of CQv1 and recommendation to update queues to v2 using policies before v1 gets removed in 4.0.

Reply:
Yes, definitely, let me edit the release notes.